File: vigridr/src/games/guessnumber/sampleclient/ClientLogic.cs
using System;
using System.Threading;
using System.Diagnostics;

public class Solution
{
    public Random random;
    public Command playTurn(WorldModel wm) {
        int guess = random.Next(10) + 1;
        Command command = new Command(guess);
        return command;
    }

    public Solution(GameInit gameInit)
    {
        random = new Random();
    }

}

File: vigridr/src/games/guessnumber/sampleclient/ClientLogic.java
import java.util.Random;

public class ClientLogic {
    public Random random;

    public ClientLogic(GameInit gameInit) {
        random = new Random();
    }

    public Command playTurn(WorldModel wm) {
        return new Command(random.nextInt(10) + 1);
    }
}
